src/components/NewCommentForm.tsx

```diff
@@ -41,28 +139,34 @@ export const NewCommentForm: React.FC = () => {
 
         <div className="control has-icons-left has-icons-right">
           <input
+            value={email}
+            onChange={handleEmailChange}
             type="text"
```

Comment:
The `type` attribute for the email input should be `email` instead of `text` to ensure proper email validation and keyboard support on mobile devices.